pandas/tests/io/test_feather.py

```diff
@@ -247,3 +248,20 @@ def test_string_inference(self, tmp_path):
             result = read_feather(path)
         expected = pd.DataFrame(data={"a": ["x", "y"]}, dtype="string[pyarrow_numpy]")
         tm.assert_frame_equal(result, expected)
+
+    def test_out_of_bounds_datetime_to_feather(self, tmp_path):
+        # GH#47832
+        df = pd.DataFrame(
+            {
+                "date": [
+                    datetime.fromisoformat("1654-01-01"),
+                    datetime.fromisoformat("1920-01-01"),
+                ],
+            }
+        )
+
+        path = tmp_path / "tmp.feather"
+        df.to_feather(path)
+        expected = df.copy()
+        result = read_feather(path)
+        tm.assert_frame_equal(result, expected)
```

Comment:
Can just do `self.check_round_trip(df)` instead, then you don't even need `tmp_path`.

Reply:
Ah I see, I've updated the test code to just use `self.check_round_trip(df)`